test/Microsoft.AspNetCore.Owin.Tests/OwinEnvironmentTests.cs

```diff
@@ -129,6 +130,23 @@ public void OwinEnvironmentSuppliesDefaultsForMissingRequiredEntries()
             Assert.Equal("1.0", env["owin.Version"]);
         }
 
+        [Fact]
+        public void OwinEnvironmentImpelmentsGetEnumerator()
+        {
+            HttpContext context = CreateContext();
```

Comment:
style: use `var` whenever possible instead of the explicit type. I know that the rest of the test class doesn't follow this guideline but it's never too late to start. 

Also, combine: `var owinEnvironment = new OwinEnvironment(CreateContext());`